gradle/libs.versions.toml

```diff
@@ -1,8 +1,8 @@
 [versions]
-kotlin = "2.1.21"
-kotlin-for-gradle-plugin = "2.1.0" # Kotlin 2.1 removes support for the used language version / api version: KT-60521
+kotlin = "2.2.0"
+kotlin-for-gradle-plugin = "2.1.0"
```

Comment:
And why do we need to keep `kotlin-for-gradle-plugin` version `2.1.0`?

Reply:
TL;DR: it has to be even lower

Reply:
aa ok, thank you, I got it 🙏